vendor/modules.txt

```diff
@@ -715,6 +715,7 @@ github.com/fatih/camelcase
 ## explicit
 github.com/fatih/color
 # github.com/form3tech-oss/jwt-go v3.2.2+incompatible
+## explicit
```

Comment:
any idea why this is here now?

Reply:
I have executed `go mod vendor`, which resulted in the above change. Generally this will update go.mod, but here both go.mod file and modules.txt file got update